Album/ViewControllers/ProfileControllers/FriendGridViewController.m

```diff
@@ -0,0 +1,79 @@
+//
+//  FriendGridViewController.m
+//  Album
+//
+//  Created by Airei Fukuzawa on 7/21/22.
+//
+
+#import "FriendGridViewController.h"
+#import "FriendProfileViewController.h"
+#import "ParseAPIHelper.h"
+#import "PhotoCollectionCell.h"
+
+@interface FriendGridViewController () <UICollectionViewDataSource, UICollectionViewDelegate>
+@property (weak, nonatomic) IBOutlet UICollectionView *friendCollectionView;
+@property (strong, nonatomic) ParseAPIHelper *apiHelper;
+@property (strong, nonatomic) NSArray *friendsArray;
+@end
+
+@implementation FriendGridViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Set API helper
+    self.apiHelper = [[ParseAPIHelper alloc] init];
+    // Assign collection view delegate and dataSource
+    self.friendCollectionView.delegate = self;
+    self.friendCollectionView.dataSource = self;
+    // Fetch friends
+    [self.apiHelper fetchFriends:self.user.objectId withBlock:^(NSArray *friendArr, NSError *error) {
+                                                        if (friendArr != nil) {
+                                                        self.friendsArray = friendArr;
+                                                        dispatch_async(dispatch_get_main_queue(), ^{
+                                                        [self.friendCollectionView reloadData];
+                                                        });
+                                                        } else {
+                                                        NSLog(@"%@", error.localizedDescription);
+                                                        }
+                                                    }];
```

Comment:
consider using indentation here to make your code more readable

Reply:
All indentation fixed!